### DISCUSSION TITLE
Setting up Fail2ban with UFW

### DISCUSSION THREAD

**Discussion by DistractedCanuck** [dani-garcia/vaultwarden#1591] [Q&A]
Good day, 

I'm trying to harden my vault using fail2ban.  I've followed the instructions on the wiki.  

However, I am currently using ufw as the fire wall for my server.  I can't seem to make the ban action work.  

The jail log fail is pointed to the external volume of my docker container.  The IP in the log is correct and the time stamp matches the server.  

Any help would be greatly appreciated!  

Thanks!

**Comment by DistractedCanuck** [comment]
I also had to change the fail2ban backend to systemd instead of auto to launch it since it couldn't find the bitwarden.log file. 

Once I did it launched and I believe it is now reading the log.

**Reply by hariseldon99** [reply]
Can you post your config files? I'm trying to do the same thing on my VPS.
